frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -170,7 +170,12 @@ export class RotkehlchenApi {
         },
         { validateStatus: validAccountOperationStatus }
       )
-      .then(handleResponse);
+      .then(value => {
+        if (value.status === 409) {
```

Comment:
I am sneaking this in, this is for the frontend to treat a 409 (no user logged in) as a successful logout for the frontend and get you to the login screen.

Reply:
I was going to ask about it :+1:

Reply:
I did some weird forced reloadings and I couldn't logout due to 409 :P